src/components/funFacts/funfacts.ts

```diff
@@ -0,0 +1,26 @@
+export interface FunFact {
+    id: number;
+    prompt: string;
+    detail: string;
+}
+
+export const funFacts: FunFact[] = [
+    {
+        id: 1,
+        prompt: "Mars is named after the Roman god of war.",
+        detail: "The month march was also named after him. Mars's moons, called Phobos and Deimos, are named after the gods of fear",
+    },
+
+    {
+        id: 2,
+        prompt: "A year on Mars is not the same as on Earth?",
+        detail: "One martian year is about twice as long as a year on Earth, so you only get one birthday every two years!",
+    },
+
+    {
+        id: 3,
+        prompt: "Did you know that gravity is different on Mars? ",
+        detail: "On Mars, you only weigh three eighths of what you weigh on Earth, meaning you could jump more than twice as high!",
+    },
+
+]
```

Comment:
Semicolon, and single blank line at end of file
```suggestion
];
```